### ISSUE TITLE
Fix markdown linting errors - "Images should have alternate text" 

### ISSUE THREAD

**Issue by spydon** [flame-engine/gamepads#26]
The markdown warning "MD045/no-alt-text Images should have alternate text (alt text)" is triggering on all the readme files.

```
packages/gamepads_darwin/README.md:9:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_darwin/README.md:12:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_darwin/README.md:15:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_darwin/README.md:18:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_darwin/README.md:97:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_darwin/README.md:103:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_darwin/README.md:112:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:9:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:12:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:15:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:18:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:97:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:103:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_linux/README.md:112:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:9:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:12:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:15:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:18:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:97:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:103:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_platform_interface/README.md:112:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:9:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:12:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:15:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:18:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:97:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:103:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads_windows/README.md:112:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:9:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:12:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:15:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:18:5 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:97:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:103:3 MD045/no-alt-text Images should have alternate text (alt text)
packages/gamepads/README.md:112:3 MD045/no-alt-text Images should have alternate text (alt text)
README.md:9:5 MD045/no-alt-text Images should have alternate text (alt text)
README.md:12:5 MD045/no-alt-text Images should have alternate text (alt text)
README.md:15:5 MD045/no-alt-text Images should have alternate text (alt text)
README.md:18:5 MD045/no-alt-text Images should have alternate text (alt text)
README.md:97:3 MD045/no-alt-text Images should have alternate text (alt text)
README.md:103:3 MD045/no-alt-text Images should have alternate text (alt text)
README.md:112:3 MD045/no-alt-text Images should have alternate text (alt text)
```
